UK/LogonDetails.py

```diff
@@ -60,9 +125,12 @@
 
 f.close
 
-# Add CID as Nickname to VCCS profiles
+# Set VCCS Settings
 
-PrfVCCS=("TeamSpeakVccs	Ts3NickName	"+CID)
+PrfVCCSNick=("TeamSpeakVccs	Ts3NickName	"+CID)
+PrfVCCSPTT=("TeamSpeakVccs Ts3G2GPtt "+PTT)
+PrfVCCSInput=("TeamSpeakVccs	CaptureDevice	"+VCCSInput)
+PrfVCCSOutput=("TeamSpeakVccs	PlaybackDevice	"+VCCSOutput)
```

Comment:
Should we be considering setting the `TeamSpeakVccs	CaptureMode` as well for completeness or does it not matter/default to something sensible?